solana/ts/sdk/ntt.ts

```diff
@@ -0,0 +1,799 @@
+import { type ChainName, toChainId, coalesceChainId, type ChainId, SignedVaa } from '@certusone/wormhole-sdk'
```

Comment:
should we provide an ez way to go from sdkv1 chain names to sdkv2 chain names?

Reply:
Imho no. The thing should be retired in the Bladerunner sense, not in the senior citizen sense.